hoomd/md/many_body.py

```diff
@@ -0,0 +1,343 @@
+    energies imposing single bond per particle condition:
+
+    .. math::
+        :nowrap:
+
+            \\begin{eqnarray*}
+            v^{\\left( 3b \\right)}_{ijk}=\\lambda \\epsilon\\,\\hat{v}^{ \\left( 2b \\right)}_{ij}
+            \\left(\\vec{r}_{ij}\\right) \\cdot \\hat{v}^{ \\left( 2b \\right)}_{ik}
+            \\left(\\vec{r}_{ik}\\right)~,\\
+            \\end{eqnarray*}
+
+    where the two body potential is rewritten as:
+
+    .. math::
+        :nowrap:
+
+            \\begin{eqnarray*}
+            \\hat{v}^{ \\left( 2b \\right)}_{ij}\\left(\\vec{r}_{ij}\\right) =
+            \\begin{cases}
+            1 & r \\le r_{min} \\\\
+            - \\dfrac{v_{ij}\\left(\\vec{r}_{ij}\\right)}{\\epsilon} & r > r_{min} \\\\
+            \\end{cases}
+            \\end{eqnarray*}
+
+    .. attention::
+
+        The RevCross potential models an asymmetric interaction between two
+        different chemical moieties that can form a reversible bond. This
+        requires the definition of (at least) two different types of particles.
+        A reversible bond is only possible between two different species,
+        otherwise :math:`v^{\\left( 3b \\right)}_{ijk}`, would prevent any bond.
+        In our example we then set the interactions for types A and B with
+        ``rev_c.params[[('A','B'),('A','B')]] = dict(sigma=0.0,n=0,epsilon=0,lambda3=0)``
+        and the only non-zero energy only between the different types
+        ``rev_c.params[('A','B')] = dict(sigma=1,n=100,epsilon=100,lambda3=1)``.
+        Notice that the number of the minority species corresponds to the
+        maximum number of bonds.
+
+
+    This three-body term also tunes the energy required for a bond swap through
+    the coefficient:- :math:`\\lambda` - *lambda3* (unitless)
+    in `S. Ciarella and W.G. Ellenbroek 2019 <https://arxiv.org/abs/1912.08569>`_
+    is explained that setting :math:`\\lambda=1` corresponds to no energy
+    requirement to initiate bond swap, while this energy barrier scales roughly
+    as :math:`\\beta \\Delta E_\\text{sw} =\\beta \\varepsilon(\\lambda-1)`.
+
+    Note:
+        Choosing :math:`\\lambda=1` pushes the system towards clusterization
+        because the three-body term is not enough to compensate the energy of
+        multiple bonds, so it may cause unphysical situations.
+
+    Use ``params`` dictionary to set potential coefficients. The coefficients
+    must be set per unique pair of particle types.
+
+    Attributes:
+        params (TypeParameter[tuple[``particle_type``, ``particle_type``], dict]):
+            The revcross potential parameters. The dictionary has the following keys:
+
+            * ``epsilon`` (`float`, **required**) - :math:`\\varepsilon` (in units of energy)
+
+            * ``sigma`` (`float`, **required**) - :math:`\\sigma` - (in distance units)
+
+            * ``n`` (`float`, **required**) - *n* - (unitless)
+
+            * ``lambda3`` (`float`, **required**) - *lambda3`* - (unitless)
```

Comment:
Check how the docs are rendering. There are some other places where I'm not sure about the syntax correctness.
```suggestion
            * ``lambda3`` (`float`, **required**) - *lambda3* - (unitless)
```

Reply:
Shouldn't `*lambda3*` be in math notation given sigma and varepsilon?